app/src/main/java/edu/agh/susgame/front/service/web/socket/webmanagers/WebGameManager.kt

```diff
@@ -18,8 +20,34 @@ class WebGameManager(
     }
 
     fun handleGameState(decodedMessage: ServerSocketMessage.GameState) {
-        gameManager.packetsReceived.value = decodedMessage.server.packetsReceived
+        gameManager.setServerReceivedPackets(decodedMessage.server.packetsReceived)
         gameManager.updatePathsFromServer(decodedMessage)
+
+        decodedMessage.players.forEach { playerDTO ->
+            gameManager.setPlayerTokens(
+                playerId = PlayerId(playerDTO.id),
+                tokens = playerDTO.currentMoney
+            )
+        }
+        decodedMessage.routers.forEach { routerDTO ->
+            gameManager.setRouterBufferSize(
+                nodeId = NodeId(routerDTO.id),
+                size = routerDTO.bufferSize
+            )
+            gameManager.setRouterCurrentPackets(
+                nodeId = NodeId(routerDTO.id),
+                packets = routerDTO.bufferSize - routerDTO.spaceLeft
+            )
+            gameManager.setRouterUpgradeCost(
+                nodeId = NodeId(routerDTO.id),
+                cost = routerDTO.upgradeCost
+            )
+        }
+        gameManager.gameStatus.value = decodedMessage.gameStatus
+    }
+
+    fun handlePathUpdate() {
+//        gameManager.updatePathsFromServer(decodedMessage)
```

Comment:
Powinno być zakomentowane?

Reply:
zrobiłem to pod sytuacje rozdzielenia zmiany sciezki do innego dto, narazie niestety nikt się tym nie zajął